src/main/java/com/blackducksoftware/ohcount4j/Ohcount.java

```diff
@@ -143,6 +149,9 @@ static void summarize(List<File> files, List<String> filenames) throws IOExcepti
 
         @Option(name = "-l", usage = "show supported languages")
         boolean supportedLanguages = false;
+
+        @Option(name = "-i", usage = "individual files line count summary")
+        boolean interactive = false;
```

Comment:
This variable would be better as 'individual' to reflect the purpose of the switch: "individual files line count detail"